src/components/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'react';
+
+import classes from './Card.module.css';
+import { CardProps } from './Card.types';
+
+const MAX_TEXT_LENGTH = 80;
```

Comment:
NIT: I prefer to have separate file for constants. Just a small separation of concerns.